spi/common/identity-trust-spi/src/main/java/org/eclipse/edc/identitytrust/model/VerifiableCredential.java

```diff
@@ -41,6 +42,8 @@ public class VerifiableCredential {
     public static final String VERIFIABLE_CREDENTIAL_PROOF_PROPERTY = "https://w3id.org/security#proof";
     private List<CredentialSubject> credentialSubject = new ArrayList<>();
     private String id; // must be URI, but URI is less efficient at runtime
+
+    @JsonProperty("type")
```

Comment:
nit: should we do the same for `credentialSubject`? i.e. call the field `credentialSubjects` (plural), but annotating with a JsonProperty to conform to the spec

Reply:
done